parts/k8s/kubernetescustomscript.sh

```diff
@@ -57,10 +57,6 @@ else
     FULL_INSTALL_REQUIRED=true
 fi
 
-if [[ $OS == $UBUNTU_OS_NAME ]]; then
```

Comment:
I think we need to get rid of the "apt-mark hold" invocation in CSE, as the hold command will wait for the release of apt locks, which may take a long time if cloud-init has already reserved them.